caluma/form/historical_schema.py

```diff
@@ -9,9 +9,9 @@
 from .schema import (
     Answer,
     DateAnswer,
-    Document,
     FileAnswer,
     FloatAnswer,
+    FormDjangoObjectType,
```

Comment:
What is this?

Reply:
This is our ObjectType we use for resolving question and answer types. It can be found[here](https://github.com/projectcaluma/caluma/blob/master/caluma/form/schema.py#L63). I was not able to inherit from the `Document` type, so I've made it analoguous to this class. We could also just use `DjangoObjectType` here but it doesn't hurt anyway.

Reply:
Alright